ProcessMaker/Http/Requests/GetAllCasesRequest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace ProcessMaker\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class GetAllCasesRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
+    public function authorize(): bool
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, \Illuminate\Contracts\Validation\ValidationRule|array<mixed>|string>
+     */
+    public function rules(): array
+    {
+        return [
+            'userId' => 'sometimes|integer',
+            'status' => 'sometimes|in:in_progress,completed',
```

Comment:
Please use uppercase for the status to keep the code style of PM, and also update it to the architecture document.

Reply:
It was fixed